test/functional/test_framework/test_node.py

```diff
@@ -206,6 +207,41 @@ def assert_debug_log(self, expected_msgs):
                 if re.search(re.escape(expected_msg), log, flags=re.MULTILINE) is None:
                     self._raise_assertion_error('Expected message "{}" does not partially match log:\n\n{}\n\n'.format(expected_msg, print_log))
 
+    def assert_start_raises_init_error(self, extra_args=None, expected_msg=None, partial_match=False, *args, **kwargs):
```

Comment:
Few more tests need fixing after `assert_start_raises_init_error` was moved to TestNode